engine/src/flutter/shell/platform/embedder/embedder.h

```diff
@@ -3565,6 +3610,7 @@ typedef struct {
   FlutterEngineUpdateSemanticsEnabledFnPtr UpdateSemanticsEnabled;
   FlutterEngineUpdateAccessibilityFeaturesFnPtr UpdateAccessibilityFeatures;
   FlutterEngineDispatchSemanticsActionFnPtr DispatchSemanticsAction;
+  FlutterEngineSendSemanticsActionFnPtr SendSemanticsAction;
```

Comment:
This needs to be appended at the end of the struct.

Reply:
Ah whoops, I missed that. Good catch!